rakelib/lint.rake

```diff
@@ -0,0 +1,28 @@
+namespace :lint do
```

Comment:
Yeah given their specifics (one not being code to compile as a framework but rather templates, the other being a CLI that we need to wrap manually) I can guess why they might be different. I just don't want to repeat code and more importantly risk any discrepancy between them or bugs being fixed in one place but not the other…

But that will probably be work for a dedicated PR in Eve once everything is merged and works then.